examples/scripts/vsft.py

```diff
@@ -0,0 +1,198 @@
+# flake8: noqa
+# Copyright 2024 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+# regular:
+python examples/scripts/vsft.py \
+    --model_name_or_path="llava-hf/llava-1.5-7b-hf" \
+    --report_to="wandb" \
+    --learning_rate=1.41e-5 \
```

Comment:
Maybe pick something a bit less magic?
```suggestion
    --learning_rate=1.00e-5 \
```

Reply:
Will do, the value is copied from the SFT example: https://github.com/huggingface/trl/blob/57aebe9c36021e679662181468b04ff42432daf3/examples/scripts/sft.py#L20